main.go

```diff
@@ -297,54 +298,78 @@ func enableDevworkspaceSupport(mgr manager.Manager) error {
 		return err
 	}
 
-	// we only enable Devworkspace support, if there is the controller.devfile.io resource group in the cluster
-	// we assume that if the group is there, then we have all the expected CRs there, too.
+	// we only enable DevWorkspace support, if there is the controller.devfile.io resource group in the cluster
+	// (we assume that if the group is there, then we have all the expected CRs there, too.)
+	// and DevWorkspaces are enabled at least on one CheCluster
```

Comment:
This would mean that we'd have to restart the operator at each "flip" of the `Spec.DevWorkspace.Enable`.